test/tab-components/frameTest.js

```diff
@@ -278,6 +279,20 @@ describe('frame tests', function () {
         .url(url)
         .waitForVisible('#viewerContainer')
     })
+
+    it('loads HTML properly despite having .pdf extension', function * () {
+      // Add custom headers for this request, so we can override the
+      // default mime type that will otherwise get added for a .pdf file.
+      let customHeaders = { 'Content-Type': 'text/html; charset=utf-8' }
+      let url = Brave.server.url('html_with_pdf_extension.pdf')
+      Brave.server.defineHeaders(url, customHeaders)
+
+      yield this.app.client
+        .tabByIndex(0)
+        .url(url)
+        .waitForUrl(url)
```

Comment:
this test needs a `.windowByUrl(Brave.browserWindowUrl)` here to switch from the page context into the Brave window context or else it fails because there's no activeTabTitle component found.

Reply:
@diracdeltas OK, I've added this in https://github.com/brave/browser-laptop/pull/13587/commits/0224cd6595b5f8df80bf5018da0c87c495f9d7e9.